nix/dapp.nix

```diff
@@ -0,0 +1,740 @@
+# This file has been generated by dapp2nix 0.0.2, do not edit.
+{ solidityPackage, solc, dapp2 }:
```

Comment:
Do you know why there's such a thing as `dapp2`?

Reply:
Nope :confused: